src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_FIGURES = 5;
+    private static final int DEFAULT_NUMBER = 10;
+    private static final String DEFAULT_COLOR = "White";
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int radius = random.nextInt(DEFAULT_NUMBER);
+        int index = random.nextInt(MAX_FIGURES);
+        String color = colorSupplier.getRandomColor();
+
+        return switch (index) {
+            case 0 -> new Circle(color, radius);

```

Comment:
```suggestion
            case 0 -> new Circle(color, your_method);
```

create separate private method for `random.nextInt(DEFAULT_NUMBER)` and use it for each type of figure